protocol/packages/currencies/src/currency_macro.rs

```diff
@@ -1,6 +1,6 @@
 pub use serde::{Deserialize, Serialize};
 
-pub use sdk::schemars::{self, JsonSchema};
+pub use sdk::schemars::JsonSchema;
```

Comment:
curious how have you discovered this ... because in other cases using `sdk::schemars` is required